app/lib/Plugins/SearchEngine/SqlSearch2.php

```diff
@@ -382,10 +382,10 @@ private function _processQueryTerm(int $subject_tablenum, $query) {
 	 	}
 	 	$ap = $field ? $this->_getElementIDForAccessPoint($subject_tablenum, $field) : null;
 	 	$words = [$term->text];
-	 	if($field && !is_array($ap)) {
-	 		array_unshift($words, $field);
-	 		$field = null;
-	 	}
+		if($field && $ap && !is_array($ap)) {
```

Comment:
_getElementIDForAccessPoint() returns null or an array, so we're effectively removing the ability to treat word1:word2 values as text when word1 is not a valid access point here. This behavior is actually needed, so if we want to support _fulltext as an access point we'll need to explicitly test for it.

Reply:
OK so something like: 
```php
<?php
		if($field && $field !== '_fulltext' && !is_array($ap)) {
			array_unshift($words, $field);
			$field = null;
		}
```